src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    protected String color;
+    protected int number;

```

Comment:
The 'number' field should also have an explicit access modifier for the same reasons as the 'color' field.